content/docs/1.3.3/references/settings.md

```diff
@@ -72,6 +73,17 @@ By default, Longhorn will block `kubectl drain` action on a node if the node con
 
 If this setting is enabled, Longhorn will not block `kubectl drain` action on a node even if the node contains the last healthy replica of a volume.
 
+#### Node Drain Policy
+
+> Default: `block-if-contains-last-replica`
```

Comment:
Please mark `Allow Node Drain with the Last Healthy Replica` is deprecated.

Reply:
I thought we agreed that we only deprecated it in Longhorn 1.5.0? What do you think?

Reply:
yeah, as discussed, there is one thing that needs clarification. 

We can deprecate a setting or function in a patch version. For example, in this case, we will deprecate it in versions 1.3.3 and 1.4.2, and it should be removed from version 1.5.0.

https://github.com/longhorn/longhorn/wiki/Deprecation-Policy

```
- Deprecate a feature in a patch or feature release. The feature should still work until it is removed later.
- After deprecation, remove the feature in the just following feature release with the upgrade migration to ensure a smooth upgrade.
```

cc @longhorn/dev

Reply:
Thank you very much. All clear now. I marked it as deprecated in v1.3.3 and v1.4.2 and will remove it in v1.5.0 by this ticket https://github.com/longhorn/longhorn/issues/5620

Reply:
Exactly @PhanLe1010 :+1: